circle/main.go

```
@@ -50,8 +59,30 @@ func main(){
+		circle.GET("/pendingcircle", controllers.PendingCircle)
+		circle.POST("/approvecircle", controllers.ApproveCircle)
+	}
+	search:=r.Group("/search")
+	{
+        search.POST("/searchcircle",controllers.SearchCircle)
+		search.POST("/searchtest",controllers.SearchTest)
+		search.GET("/searchhistory",controllers.SearchHistory)
+		search.GET("/deletehistory",controllers.DeleteHistory)
+		search.POST("/searchpractice",controllers.SearchPractice)
 	}
 	r.Run("0.0.0.0:8080")
```

golangci-lint (1.62.2)

Error return value of `r.Run` is not checked

(errcheck)